bigquery_etl/udf/publish_udfs.py

```diff
@@ -134,6 +136,14 @@ def publish_udf(
             fr'library = "gs://{gcs_bucket}/{gcs_path}\1"', definition
         )
 
+        # add UDF descriptions
+        if raw_udf.filepath not in SKIP:
+            escaped_description = json.dumps(str(raw_udf.description))
```

Comment:
The descriptions need to be escaped since otherwise quotation marks and other characters will make the query invalid. Converting it to JSON seemed like the easiest thing to do here

Reply:
BQ has triple-quoted strings. Can we wrap the description in `"""` instead of json-ifying? It seems like that would be slightly safer.

Reply:
Json-ifying also handles escaping of some other characters, for example the `\x01` in `[...]SELECT MAX(n)     FROM UNNEST(GENERATE_ARRAY(0, 8 * BYTE_LENGTH(b)))
  AS n     WHERE BIT_COUNT(SUBSTR(b >> n, -1) & b''\x01'') > 0));  See also: bits_to_days_since_seen.sql[...]`. Just wrapping the description in `"""` will result in a `re.error: bad escape \x at position` error for these cases when used in `OPTIONS_RE.sub`.

Reply:
Sounds worth adding a comment to this line. 👍